packages/account-v2/src/modules/FinancialAssessmentFields/__tests__/FinancialAssessmentFields.spec.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { APIProvider, AuthProvider, useAuthorize } from '@deriv/api-v2';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { FinancialAssessmentFields } from '../FinancialAssessmentFields';
+
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn(() => ({ isMobile: false })),
+}));
+
+jest.mock('@deriv/api-v2', () => ({
+    ...jest.requireActual('@deriv/api-v2'),
+    useAuthorize: jest.fn(() => ({
+        data: {
+            landing_company_name: 'svg',
+        },
+    })),
+}));
+
+beforeEach(() => {
+    jest.restoreAllMocks();
+});
+
+const initialValues = {
+    accountTurnover: '',
+    educationLevel: '',
+    employmentIndustry: '',
+    employmentStatus: '',
+    estimatedWorth: '',
+    incomeSource: '',
+    netIncome: '',
+    occupation: '',
+    sourceOfWealth: '',
+};
+
+describe('FinancialAssessmentFields', () => {
+    it('renders all form fields', () => {
+        render(
+            <APIProvider>
+                <AuthProvider>
+                    <Formik initialValues={initialValues} onSubmit={jest.fn()}>
+                        <Form>
+                            <FinancialAssessmentFields />;
+                        </Form>
+                    </Formik>
+                </AuthProvider>
+            </APIProvider>
```

Comment:
let's move the <APIProvider /> and <AuthProvider /> out to a const returning function

```suggestion
            const renderComponent = (children: React.ReactNode) => {
            
            return (
            <APIProvider>
                <AuthProvider>
                    {children}
                </AuthProvider>
            </APIProvider>
            )
            }
```

Reply:
Updated.